__mocks__/mapDefinitions/FullTranscriptMapDefinition.ts

```diff
@@ -1,144 +1,16 @@
-﻿export const fullTranscriptMapDefinitionString = `
```

Comment:
This change might be an accident?

Reply:
nope I was using it for testing, fixed now